test/use-swr-error.test.tsx

```diff
@@ -37,7 +37,7 @@ describe('useSWR - error', () => {
         { onError: (_, errorKey) => (erroredSWR = errorKey) }
       )
       if (error) return <div>{error.message}</div>
-      return <div>hello, {data}</div>
+      return <div><>hello, {data}</></div>
```

Comment:
why do we adding Fragment here?

Reply:
<img width="1148" alt="Screen Shot 2022-05-13 at 07 34 10" src="https://user-images.githubusercontent.com/15878786/168188285-ccc9b92d-2d38-43d6-94fc-425223231b6f.png">


Because the data is infered as Error here.